api/src/services/taxonomy-service.ts

```diff
@@ -192,4 +192,47 @@ export class TaxonomyService {
 
     return response ? this._sanitizeSpeciesData(response.hits.hits) : [];
   }
+
+  private formatEnrichedData = (data: SearchHit<any>) => {
```

Comment:
I believe making service methods private makes mocking the method more difficult or even impossible in the service tests. I think we should prefix this method with an `_` and make it public, like other methods in the class

Reply:
Also, is it possible to be more specific with the typing on the `data` parameter? It should be `SearchHit<ITaxonomySource>` although I could be wrong